Web/Edubase.Web.UI/package.json

```diff
@@ -41,13 +42,14 @@
     "stylelint-config-standard": "^20.0.0",
     "stylelint-scss": "^3.18.0",
     "stylelint-webpack-plugin": "^2.1.1",
-    "vue": "^2.6.11",
-    "vue-loader": "^15.9.1",
-    "vue-template-compiler": "^2.6.11",
-    "webpack": "^5.75.0",
+    "terser-webpack-plugin": "^5.3.10",
+    "vue": "^3.4.37",
```

Comment:
We have many dependencies in the project, since returning from hols there are several NPM packages ready to be updated again. Like you say, maybe look at the minor/ patch updates in a separate ticket later - We could go around in circles.. spend our time solving issues that may stem from bugs in a minor update, and by the time we resolve those, more updates are available